### DISCUSSION TITLE
Azuracast on subdmain.domain.com and Worpress on domain.com on same server.

### DISCUSSION THREAD

**Discussion by pwrj** [AzuraCast/AzuraCast#5369] [Q&A]
Please show me an easy way to run Wordpress on mydomain.com and Azuracast on a subdomain like radio.mydomain.com - I currently have Azuracast setup and running on radio.mydomain.com using standart port 80, 443. Now, I just need to install Wordpress on mydomain.com to run my station's website. I follow the instruction https://docs.azuracast.com/en/administration/docker/multi-site-installation but it didn't work for me, perhaps I did something wrong, who knows?

**Comment by Vicinis** [comment]
For simplicity, it’s best to have 2 ip’s. Make sure the 2nd ip has no web sever running on it and put azuracast on that.

**Reply by pwrj** [reply]
Two IPs on the same server?

**Reply by Vicinis** [reply]
Yes. Or different servers. Up to you. 
By using the same ip trying to run 2 web servers (on same server) is difficult. 
get a spare ip with no web server running on it and run your azuracast on that ip and point that ip to your radio.mydomain.com
I myself have 4 servers. 
1 for website with Wordpress domain.com
3 for azuracast *.domain.com

**Reply by pwrj** [reply]
@Vicinis Got you.

**Reply by Vicinis** [reply]
This isn’t me advertising anything. Just saying VPC’s are cheap now. My 4 servers are total $33 a month and each station is its own server.

**Reply by pwrj** [reply]
@Vicinis 4 Servers for $33.00/Month, please do tell.

**Reply by Vicinis** [reply]
@pwrj 
since this is going off topic from azuracast, you can email me at 
github @ cricket.radio
i'll tell you my setup

**Reply by pwrj** [reply]
@Vicinis Will do. Thanks!

**Comment by promildtech** [comment]
If anyone is still looking for a way to do this, we can help

**Comment by Heimatradio** [comment]
if you are using one linux vps with one IP you can do the following steps:
1. Install the CloudPanel from https://www.cloudpanel.io/docs/v2/getting-started/other/  it's a free panel!!!!!!!!!!!
  The Panel is accesable via IP:8443
2. Install Azuracast and change the standard-ports http to 9080, https to 9443 and sftp to 9022, Station-ports best 10000-10999
3. Create 2 websites in the CloudPanel
    1. with Domain for your Wordpress-Website
    2. with subdomain for your Azuracast, use Reverse Proxy and set Reverse-Proxy-Url to http://127.0.0.1:9080
 4. Create free Let's Encrypt certificates for both website
Finally your website is visitable in https at domain.com
The Azuracast instance is visitable at subdomain.domain.com

In my setup is https://azuracast.stream-online.eu the AzuraCast
and https://cloud.stream-online.eu my website
